platform/accessibility/common-issues/focus-order-guidelines.md

```diff
@@ -0,0 +1,70 @@
+# [KEYBOARD]: Focus order MUST be visually and programmatically usable
```

Comment:
is this the title of the document?

Reply:
I changed the document title to match the file name in the commit I'm making in a short.